independent-publisher-connectors/Rarible/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3342 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v0.1",
+    "title": "Rarible",
+    "description": "Rarible Multichain Protocol is a blockchain-agnostic and decentralized tool to query, issue, and trade NFTs.",
+    "contact": {
+      "name": "Roy Paar",
+      "email": "roypaar@microsoft.com"
+    }
+  },
+  "host": "ethereum-api.rarible.org",
+  "basePath": "/v0.1",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/erc20/balances/{contract}/{owner}": {
+      "get": {
+        "description": "Return ERC20 token balance",
+        "summary": "getErc20Balance",
+        "tags": [
+          "erc20-balance-controller"
+        ],
+        "operationId": "getErc20Balance",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "contract",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Address of the contract",
+            "x-ms-summary": "Address of the contract",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "owner",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Address of the ERC20 token owner",
+            "x-ms-summary": "Address of the ERC20 token owner",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Erc20DecimalBalance"
+            },
+            "headers": {}
```

Comment:
@RoyPaarMS Empty property values in the swagger can be removed until unless you have a specific reason to keep them.